python/tvm/relax/transform/lazy_transform_params.py

```diff
@@ -118,9 +119,15 @@ class LazyTransformParamsMutator(PyExprMutator):
         The module to be transformed
     """
 
-    def __init__(self, mod: IRModule = None) -> None:
+    def __init__(
+        self, fget_item, fset_item, get_item_param, set_item_param, mod: IRModule = None
+    ) -> None:
         super().__init__(mod)
         self.mod = mod
+        self.fget_item = fget_item
+        self.get_item_param = get_item_param
```

Comment:
What is the purpose of the `get_item_param` and `set_item_param`?  I don't see any unit test with their intended use.  Since the same parameters are forwarded to all `get_item` and `set_item` calls, it seems the same effect could be achieved by having a stateful callback, rather than passing an argument which is then passed right back to you.

Reply:
My bad. I forget to add a test on this. The case is that I need to call `get_item(loader, index)`. Since shard loader is created at runtime, it cannot be a stateful callback and I have to modify the interface of `get_item`. Does this make sense to you?